fiftyone/core/view.py

```diff
@@ -439,6 +439,119 @@ def _make_view_stages_str(self):
             ]
         )
 
+    def first(self):
```

Comment:
Unfortunately we can't use `_get_first()` for views because they may contain sorting operations of their own.

For views, I can't think of anything we can easily do that's better than retaining the existing implementation.